releases/cosmwasm/2025-01-ITS-v1.1.0.md

```diff
@@ -28,7 +28,7 @@ Changes in this release:
 Upload new ITS Hub contract
 
 ```bash
-node cosmwasm/submit-proposal.js store -c InterchainTokenService -t "Upload InterchainTokenService contract v1.1.0" -d "Upload InterchainTokenService contract v1.1.0" -r $RUN_AS_ACCOUNT --deposit 2000000000 -a ./wasm
+node cosmwasm/submit-proposal.js store -c InterchainTokenService -t "Upload InterchainTokenService contract v1.1.0" -d "Upload InterchainTokenService contract v1.1.0" -r $RUN_AS_ACCOUNT --deposit 2000000000 --instantiateAddresses $INIT_ADDRESSES -a ./wasm
```

Comment:
Where is `INIT_ADDRESSES` defined? Create a params table. This wasn't added during devnet deployment, is that a problem? And why can't this set automatically via the script, discuss with Edwin?

Reply:
Same for RUN_AS_ACCOUNT, add to the table

Reply:
Looks like `INIT_ADDRESSES` was passed in during deployment but not documented.  

Added the table above.